tests/unittests/graphOptzTest.cpp

```diff
@@ -2152,3 +2152,40 @@ TEST_F(GraphOptz, dceBeforeOptimizeTranpose) {
   // transpose should be eliminated and replaced by the transposed constant.
   EXPECT_TRUE(llvm::isa<Constant>(save1->getInput()));
 }
+
+TEST_F(GraphOptz, sinkTransposeBelowChannelShuffleNodes) {
```

Comment:
nit: Doxygen comments on the purpose of the test.

Reply:
This description isn't fully accurate -- it's testing that the `Transpose-ChannelShuffle-Transpose` sequence can be optimized to remove both transposes, by sinking the first Transpose.